test/mason/mason-test/exitCodeTest.chpl

```diff
@@ -0,0 +1,16 @@
+use MasonTest;
+use MasonUtils;
+use Spawn;
+
+
+proc checkExitStatus(cmd: string) {
+  var splitCmd = 'mason test'.split();
+  var p = spawn(splitCmd);
+  p.wait();
+  if p.exit_status == 1 {
+    exit(0);
+  }
```

Comment:
I didn't check out to test, but: If the exit_status was not 1, wouldn't this test still pass since it would fall out of the end of the program which is implicitly like an `exit(0)`?  You could add `} else { exit(1); }`, but I'd probably take the approach of having the conditional do `writeln("Got exit status 1 as expected');` and then put that string in the .good file to make it a bit more bulletproof and rely less on the subtlety of exit codes...